### PR TITLE
[DOCS] Update descs for custom expectations

### PR DESCRIPTION
**Pull request by rreinold**
I updated the descriptions to be more concise, and fixed typo

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Review by kwcanuck — APPROVED** [review]

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
-  <LinkCard topIcon label="Create a Custom Parameterized Expectation" description="Create a Custom Parameterized Expectation" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_parameterized_expectations" icon="/img/custom_expectation_icon.svg" />
-  <LinkCard topIcon label="Add auto-initializing framework support to a Custom Expectation" description="Use auto-initializing Expectations to automate Expectation parameter estimation" href="/docs/guides/expectations/creating_custom_expectations/how_to_add_support_for_the_auto_initializing_framework_to_a_custom_expectation" icon="/img/custom_expectation_icon.svg" />
-</LinkCardGrid>
\ No newline at end of file
+  <LinkCard topIcon label="Create a Custom Column Aggregate Expectation" description="evaluates a single column and produces an aggregate Metric" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Column Aggregate Expectation" description="Evaluates a single column and produces an aggregate Metric" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Column Map Expectation" description="evaluates a single column and performs a yes or no query on every row in the column" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Column Map Expectation" description="Evaluates a single column and performs a yes or no query on every column row " href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 3]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Batch Expectation" description="evaluates an entire Batch, and answers a semantic question about the Batch" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Batch Expectation" description="Evaluates an entire Batch, and answers a semantic question about the Batch" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 4]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Column Pair Map Expectation" description="evaluates a pair of columns and performs a yes or no query about the row-wise relationship between the two columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_pair_map_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Column Pair Map Expectation" description="Evaluates a pair of columns and performs a yes or no query about the row-wise relationship between the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_pair_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 5]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Multicolumn Map Expectation" description="evaluates a set of columns and performs a yes or no query about the row-wise relationship between the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_multicolumn_map_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Multicolumn Map Expectation" description="Evaluates a set of columns and performs a yes or no query about the row-wise relationship between the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_multicolumn_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 3 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 2 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 1 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 5 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 6]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Regex-Based Column Map Expectation" description="evaluates a single column and performs a yes or no regex-based query on every row in the column" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_regex_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Regex-Based Column Map Expectation" description="Evaluates a single column and performs a yes or no regex-based query on every column row" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_regex_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 7]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Set-Based Column Map Expectation" description="evaluates a single column and determines if each row in the column belongs to the specified set" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_set_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Set-Based Column Map Expectation" description="Evaluates a single column and determines if each row in the column belongs to the specified set" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_set_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 6 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 8]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Query Expectation" description="runs Expectations against custom query results and makes intermediate queries to your database" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Query Expectation" description="Runs Expectations against custom query results and makes intermediate queries to your database" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 7 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 8 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: capitalized all first letters in 0dc8a25de

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 9]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Create a Custom Parameterized Expectation" description="inherit from existing expectations to create a new customized Expectation" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_parameterized_expectations" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Create a Custom Parameterized Expectation" description="Inherits classes from existing Expectations and then creates a new customized Expectation" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_parameterized_expectations" icon="/img/custom_expectation_icon.svg" />
```

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md [hunk 10]
```diff
@@ -12,14 +12,14 @@ import LinkCard from '/docs/components/LinkCard';
...
+  <LinkCard topIcon label="Add auto-initializing framework support to a Custom Expectation" description="automate Expectation parameter estimation" href="/docs/guides/expectations/creating_custom_expectations/how_to_add_support_for_the_auto_initializing_framework_to_a_custom_expectation" icon="/img/custom_expectation_icon.svg" />
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]:
```suggestion
  <LinkCard topIcon label="Add auto-initializing framework support to a Custom Expectation" description="Automates Expectation parameter estimation" href="/docs/guides/expectations/creating_custom_expectations/how_to_add_support_for_the_auto_initializing_framework_to_a_custom_expectation" icon="/img/custom_expectation_icon.svg" />
```

**Comment by rreinold** [comment]
Looks great, can we merge?

File: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md (hunk 9 above)
Comment (rreinold) [reply docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md]: updated in 3eef32bef

**Pull request merged**